### PR TITLE
Miscellaneous fixes and improvements

### PR DESCRIPTION
**Pull request by tjanez**

### CODE REVIEW

File: tests/test-cli.sh [hunk 1]
```diff
@@ -21,6 +21,7 @@ ${OASIS_REGISTRY} verify
 
 # Create new entity metadata.
 ${OASIS_REGISTRY} entity update \
+    --assume_yes \
```
Comment (ptrus) [inline tests/test-cli.sh:24]: indentation doesn't seem right here (and bellow)

File: README.md [hunk 2]
```diff
@@ -1,12 +1,144 @@
...
+save it as a JSON file, e.g. `entity-metadata.json`, and run:
+
+```sh
+./oasis-registry/oasis-registry entity update \
+  <SIGNER-FLAGS> \
+  entity-metadata.json
+```
+
+where `<SIGNER-FLAGS>` are replaced by the appropriate signer CLI flags for your
+signer (e.g. Ledger-based signer, File-based signer).
```
Comment (ptrus) [inline README.md:52]: should there be a note here that ledger doesn't yet support singing metadata-registry statements

**Review by ptrus — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tjanez) [reply README.md:52]: :slightly_smiling_face: Yeah, good point. I'll add that.

File: tests/test-cli.sh (hunk 1 above)
Comment (tjanez) [reply tests/test-cli.sh]: Nice catch, this was due to mixture of tabs and spaces.

Comment (tjanez) [reply tests/test-cli.sh]: I've added Shell code formatting and verification via the [shfmt tool](https://github.com/mvdan/sh#shfmt)

File: README.md (hunk 2 above)
Comment (tjanez) [reply README.md:55]: Added.

**Pull request merged**